app/routes.test.js

```diff
@@ -105,19 +148,37 @@ describe('routes', () => {
   });
 
   describe('GET /organisations/:orgId', () => {
+    it('should redirect to the login page if the user is not logged in', () => (
+      checkAuthorisedRouteNotLoggedIn('/organisations/org1')
+    ));
+
+    it('should show the error page indicating the user is not authorised if the user is logged in but not authorised', () => (
+      checkAuthorisedRouteWithoutClaim('/organisations/org1')
+    ));
+
     it('should return the correct status and text', () => (
```

Comment:
`should return the correct status and text when the user is logged in and authorised` to be consistent with the rest of the tests (for all the endpoints)

Reply:
Good catch. I'm going to change this to 
`should return the correct status and text when the user is authorised`
the `logged in` bit should be implied from the authorised bit.